### ISSUE TITLE
PDFVierer Zoom OOM

### ISSUE THREAD

**Issue by wty1030** [syncfusion/flutter-widgets#606]
Hello, author. I use the [19.4.53 beta] version of pdfviewer. As long as I zoom, I will oom.my pdf only have 2 pages, Only a few dozen characters

**Comment by deepika134** [comment]
Hi wty1030,

We tried to reproduce the reported issue with provided details, but the issue couldn't be reproduced on our end. kindly share the following details to reproduce the issue on our end,

1. Platforms in which the issue is reproduced (Android, iOS, Web, macOS).
2. Device specification.
1. Issue reproducing steps.
2. Issue reproducing document.
4. Flutter SDK version.

This information will be helpful for us to provide you with a prompt solution at the earliest.

Regards,
Deepika Ravi

**Comment by wty1030** [comment]
作者您好，我使用的是手机是小米11，系统是android版本是12，内存8g，flutter的SDK版本是2.5.0，复现步骤是，打开一个文字版PDF，手指放大，上下滑动换页，它就会只显示出刚刚缩放的页，估计其他页面加载失败了，继续滑动就会崩溃。
author Hello, I am using mobile phone is Xiaomi 11. The system is the Android version 12. Memory 8G, FLUTTER's SDK version is 2.5.0, the recurring step is, open a text version PDF, Manual zoom in, and the upper and lower slides will only display the page just scaled, and it is estimated that other pages have failed, and the sliding will crash.

---Original---
From: ***@***.***&gt;
Date: Mon, Feb 28, 2022 21:07 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [syncfusion/flutter-widgets] PDFVierer Zoom OOM (Issue #606)




 
Hi wty1030,
 
We tried to reproduce the reported issue with provided details, but the issue couldn't be reproduced on our end. kindly share the following details to reproduce the issue on our end,
  
Platforms in which the issue is reproduced (Android, iOS, Web, macOS). 

Device specification. 

Issue reproducing steps. 

Issue reproducing document. 

Flutter SDK version. 
 
This information will be helpful for us to provide you with a prompt solution at the earliest.
 
Regards,
 Deepika Ravi
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by deepika134** [comment]
Hi wty1030,

We tried to reproduce the reported issue with provided details, but the issue couldn't be reproduced on our end. For your reference, we have prepared a sample (along with the output screen record of the same) and it can be downloaded from the following link,
https://www.syncfusion.com/downloads/support/directtrac/general/ze/Output-941174477.zip 

If you still reproduce the reported issue in your use case, then kindly share the modified issue reproducing in the sample. Which will be helpful for us to provide a prompt solution at the earliest.

Regards,
Deepika Ravi
